rojak-ui-app/RojakAndroid/app/src/main/java/id/artificialintelligence/rojak/models/PairOfCandidates.java

```diff
@@ -0,0 +1,103 @@
+package id.artificialintelligence.rojak.models;
+
+public class PairOfCandidates {
+    private int id;
+    private String name;
+    private String websiteUrl;
```

Comment:
This wouldn't work since the field name from the API is `website_url`.
You have to change it to `website_url` or use `@SerializedName("website_url")`

This also apply to others field name that doesn't match.